### PR TITLE
docs: add missing git clone

### PR DESCRIPTION
**Pull request by fncolon**
## Description

Add Missing Git Clone In README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +48,7 @@ For any improvements on the website, we should continuously measure its impact o
 First, install dependencies and fetch data to your local machine
 
 ```bash
+git clone https://github.com/kawalcovid19/wargabantuwarga.com.git
```
Comment (resir014) [inline README.md:51]:
```suggestion
First, clone the repository,

```bash
git clone https://github.com/kawalcovid19/wargabantuwarga.com.git
```

Then, install dependencies and fetch data to your local machine

```bash
```

I think it's better to put this on a separate step. Makes no sense if installing deps also includes the cloning step.

**Review by resir014 — CHANGES_REQUESTED** [review]
To add this, I think we need to restructure the getting started section a bit, like this:

File: README.md [hunk 2]
```diff
@@ -48,6 +48,7 @@ For any improvements on the website, we should continuously measure its impact o
...
 cd wargabantuwarga.com
```
Comment (resir014) [inline README.md:52]:
```suggestion
First, clone the repository.

```bash
git clone https://github.com/kawalcovid19/wargabantuwarga.com.git
```

Then, install dependencies and fetch data to your local machine

```bash
cd wargabantuwarga.com
```

**Review by resir014 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (fncolon) [reply README.md:52]: oohh i c i will modify it

Comment (fncolon) [reply README.md:52]: sorry for the mistake

**Comment by andriawan** [comment]
we might need extra step related to fork mechanism as i found one of contributors directly push to the main/upstream repo. you could add more explanation in term of forking mechanism to make fundamental step more obvious @fncolon

**Comment by fncolon** [comment]
@resir014 restructure, done.

**Comment by fncolon** [comment]
> we might need extra step related to fork mechanism as i found one of contributors directly push to the main/upstream repo. you could add more explanation in term of forking mechanism to make fundamental step more obvious @fncolon

maybe we can create new issue about this, and create new PR to discuss about this further

File: README.md [hunk 3]
```diff
@@ -45,15 +45,20 @@ For any improvements on the website, we should continuously measure its impact o
 
 ## Getting Started
 
-First, install dependencies and fetch data to your local machine
+First, clone the repository
```
Comment (resir014) [inline README.md:48]:
```suggestion
First, clone the repository.

```

Nitpick: double-line separator for proper paragraphs, and period at the end of sentence.

File: README.md [hunk 4]
```diff
@@ -45,15 +45,20 @@ For any improvements on the website, we should continuously measure its impact o
...
+```bash
+git clone https://github.com/kawalcovid19/wargabantuwarga.com.git
+```
+
+Then, install dependencies and fetch data to your local machine
```
Comment (resir014) [inline README.md:53]:
```suggestion
Then, install dependencies and fetch data to your local machine.
```

Nitpick: period at the end of sentence.

**Review by resir014 — CHANGES_REQUESTED** [review]

**Comment by fncolon** [comment]
@resir014 useful nitpicks, done.

**Review by resir014 — APPROVED** [review]

**Pull request merged**